src/components/ModalProduct.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import {
+  MDBBtn,
+  MDBModal,
+  MDBModalDialog,
+  MDBModalContent,
+  MDBModalHeader,
+  MDBModalTitle,
+  MDBModalBody,
+  MDBModalFooter,
+} from 'mdb-react-ui-kit';
+
+export default function ModalProduct() {
+  const [scrollableModal, setScrollableModal] = useState(true);
+
+  return (
+      <MDBModal className="custom-modal" show={scrollableModal} setShow={setScrollableModal} tabIndex='-1'>
+        <MDBModalDialog scrollable>
+          <MDBModalContent>
+            <MDBModalHeader>
+              <MDBModalTitle>Modal title</MDBModalTitle>
+              <MDBBtn
+                className='btn-close'
+                color='none'
+                onClick={() => setScrollableModal(!scrollableModal)}
+              ></MDBBtn>
+            </MDBModalHeader>
+            <MDBModalBody>
+              <img
+                src={"http://assets.myntassets.com/v1/assets/images/4332083/2018/3/23/11521799696407-Vastramay-Men-Beige-Cotton-Only-Kurta-6191521799696215-1.jpg"}
```

Comment:
@rahulmeghwal In general (not necessarily for this project) do you think you're run into cross-origin/cross-https problems if this parent site was secured but it was referencing some unsecured assets like images?

Reply:
The referenced file was unrelated and hence I removed it. Regrading the question, It is possible to run into cross-origin/cross-HTTPS .
1). Browsers may block or warn about mixed content when a HTTPS page reference a unsecured HTTP content. 
2). Loading unsecured assets from a different origin can trigger cross-origin security restrictions. 
Hence its recommended to ensure that assets(including images) are served securely over HTTPS. Given I used the image assets from another website, this may not be an issue for the original source since the images are from the same origin for their respective website.